models/comment.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require("mongoose");
+
+let comment = new mongoose.Schema({
+  desc: String,
+  date: { type: Date, default: Date.now },
+  user: { type: mongoose.Schema.Types.ObjectId, ref: "user" },
+});
```

Comment:
How you're gonna know the corresponding post of a particular comment?

Reply:
Got the point
added post: { type: mongoose.Schema.Types.ObjectId, ref: "posts" }.